MobiusTest/Source/FirstMatchers.swift

```diff
@@ -53,7 +53,8 @@ public func hasModel<Model: Equatable, Effect>(
     return { (first: First<Model, Effect>) in
         if first.model != expected {
             return .failure(
-                message: "Expected model to be <\(expected)>, got <\(first.model)>",
+                message: "Expected model to be different (−) than what got (+): \n" +
```

Comment:
```suggestion
                message: "Different model than expected (−), got (+): \n" +
```
The current message isn't grammatically correct. Something like the suggested version is also a bit shorter.